inference-engine/src/multi_device/multi_device_exec_network.cpp

```diff
@@ -495,11 +565,10 @@ InferenceEngine::Parameter MultiDeviceExecutableNetwork::GetConfig(const std::st
 InferenceEngine::Parameter MultiDeviceExecutableNetwork::GetMetric(const std::string &name) const {
     if (_workModeIsAUTO) {
         // fixme: should we wait actual device? meanwhile it will block inference, how to fix?
-        if (_alreadyActualNetwork) {
-            WaitActualNetworkReady();
-            return _networkActualNeeded->GetMetric(name);
+        if (_loadContext[ACTUALDEVICE].isAlready) {
```

Comment:
Is this covered by new tests?

Reply:
except this mock test, there is also function test, function test should cover this API, but not 100% sure.
we will check code coverage